docs/modules/release-notes/pages/all-releases/2_14.adoc

```diff
@@ -18,7 +18,7 @@ Only available for in-house devices. To enable this feature, mailto:support@kobi
 * *Bypass the passcode on iOS devices*: Devices or applications that are encountered the passcode for security or any specific testing purpose, Kobiton will help you to automatically input the passcode and continue your test execution.
 
 [NOTE]
-Worked for both manual & automation—Supported from iOS version 10.x to 13.2.3—required to strictly follow the https://support.kobiton.com/manual-testing/ios-passcode-guidelines/[guidelines] for the setup.
+Worked for both manual & automation—Supported from iOS version 10.x to 13.2.3—required to strictly follow the guidelines for the setup.
```

Comment:
Slight revision:
The bypass feature supports both manual and automated tests on iOS devices (versions 10 to 13.2.3) that have passcodes enabled following Kobiton's guidelines.